<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>270</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Placeholder.Settings.DialogTitle</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>230</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>220</width>
     <height>20</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Placeholder.Settings.TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>621</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>440</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>440</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>file name</string>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>60</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>140</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Placeholder.Settings.BrowseLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>59</number>
    </property>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Placeholder.Settings.Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Placeholder.Settings.Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
